tools/PI/DevHome.PI/Views/PrimaryWindow.xaml.cs

```diff
@@ -36,7 +36,7 @@ public sealed partial class PrimaryWindow : WindowEx
     public PrimaryWindow()
     {
         InitializeComponent();
-        ExternalToolsHelper.Instance.Init();
+        Application.Current.GetService<ExternalToolsHelper>().Init();

```

Comment:
Can we merge Init() to the ExternalToolsHelper constructor?

Reply:
Yep.